webaverse.js

```diff
@@ -41,6 +41,357 @@ import WebaWallet from './src/components/wallet.js';
 // const leftHandOffset = new THREE.Vector3(0.2, -0.2, -0.4);
 // const rightHandOffset = new THREE.Vector3(-0.2, -0.2, -0.4);
 
+window.isStart = false;
```

Comment:
We should not hack `window` like this. It's ok for testing but we should clean it up before merging.

Reply:
Ok, I'll refactor.